modules/apps/blogs/blogs-web/.npmbundlerrc

```diff
@@ -1,6 +1,5 @@
 {
     "ignore": [
```

Comment:
You should be able to remove this whole file now, since `config.js` is also gone and there's nothing to ignore anymore! 🎉